apps/microtvm/arduino/template_project/src/example_project/platform.c

```diff
@@ -28,6 +28,7 @@ static uint8_t g_aot_memory[WORKSPACE_SIZE]
     __attribute__((aligned(TVM_RUNTIME_ALLOC_ALIGNMENT_BYTES)));
 tvm_workspace_t app_workspace;
 
+// Called by TVM when an internal invariant is violated, and execution cannot continue.
 // Blink code for debugging purposes
 void TVMPlatformAbort(tvm_crt_error_t error) {
```

Comment:
Should we make these function declarations weak in Arduino, since they're weak in Zephyr?

Reply:
done.